### DISCUSSION TITLE
Limit concurrency on machines updated in parallel

### DISCUSSION THREAD

**Discussion by velddev** [kube-hetzner/terraform-hcloud-kube-hetzner#1219] [Ideas]
### Description

It seems like if there's a ".server will be updated in-place" terraform action, it will often try to do all of them in parallel. Sometimes this can cause a restart which causes longhorn to mark all volumes as tainted.

**Comment by mysticaltech** [comment]
@velddev Interesting, thanks for pointing that out. If you found a fix, PR welcome.  @kube-hetzner/core  FYI on the above.

**Comment by mysticaltech** [comment]
Normally the default `kured_options` are now better and will not allow this to happen, but let me know if it still happens to you in a new issue.
